### ISSUE TITLE
[Actua Golf] Not start

### ISSUE THREAD

**Issue by corvusd** [devmiyax/yabause#667]
Only show start credits screen. Checked whit all combinations possible: Bios and SH2 interpreter. In yabasanshiro-2.9.0-69dcb5.zip 2019/10/21.

**Comment by devmiyax** [comment]
Infinity Loop 

```
0x0604F92A: bt 0x0604F926
0x0604F928: tst r1, r1
0x0604F926: mov.l @r10, r1
0x0604F92A: bt 0x0604F926
0x0604F928: tst r1, r1
0x0604F926: mov.l @r10, r1
```

**Comment by devmiyax** [comment]
Infinity loop on HBlank

```
0x0605876E: add #0x24, r8
0x0605876C: bf/s 0x0605869C
0x0605876A: tst r7, r7
0x06058768: mov.l @r15, r7
0x06058766: mov.l r7, @(0x004, r15)
0x06058764: add #0x24, r7
0x06058762: add #0x24, r9
0x06058760: mov.l @(0x004, r15), r7
0x0605875E: mov.l r7, @r15
0x0605875C: add #0xFF, r7
0x0605875A: add #0x24, r11
0x06058758: mov.l @r15, r7
0x06058756: add #0x24, r12
0x06058754: add #0x24, r10
0x06058752: add #0x24, r13
0x06058750: add #0x24, r14
0x060586A6: mov #0x04, r1
0x060586A4: bt/s 0x06058750
0x060586A2: tst r1, r1
0x060586A0: and r0, r1
0x0605869E: mov #0x01, r1
0x0605869C: mov.w @r8, r0
```

**Comment by devmiyax** [comment]
0603AA12 <-> 0603AA4A infinity loop

**Comment by devmiyax** [comment]
3190 * 156180629 = 00000074 000000AE
14630 * 156180629 = 00000214 0000031E
3190 * 156180629 = 00000074 000000AE
14630 * 156180629 = 00000214 0000031E
0 * 156180629 = 00000000 00000000
0 * 156180629 = 00000000 00000000
0 * 156180629 = 00000000 00000000
12205 * 156180629 = 000001BB D1745FB1
11843 * 156180629 = 000001AE A7904CFF
20800 * 156180629 = 000002F4 5D174A40
25792 * 156180629 = 000003A9 E412A3C0
11843 * 156180629 = 000001AE A7904CFF
12205 * 156180629 = 000001BB D1745FB1
0 * 1162389643 = 00000000 00000000
0 * 1160932197 = 00000000 00000000
0 * -1974328039 = 00000000 00000000
0 * 1162389643 = 00000000 00000000
0 * 1160932197 = 00000000 00000000
0 * -1974328039 = 00000000 00000000
R[1]=00000000,R[2]=00000000
R[3]=00000000,R[0]=00000009

**Comment by devmiyax** [comment]
Fixed!

![Screenshot from 2020-07-11 16-01-44](https://user-images.githubusercontent.com/1174152/87218685-e51f6100-c38f-11ea-9075-e5894c4e0249.png)
![Screenshot from 2020-07-11 16-01-52](https://user-images.githubusercontent.com/1174152/87218687-e6e92480-c38f-11ea-9f19-62d2904a7c94.png)
![Screenshot from 2020-07-11 16-01-53](https://user-images.githubusercontent.com/1174152/87218688-e94b7e80-c38f-11ea-967c-fbe9131354c0.png)

**Comment by BenjaminSiskoo** [comment]
:o great

**Comment by devmiyax** [comment]
fixed at 1.8.0
